DEV_GUIDE.md

```diff
@@ -0,0 +1,106 @@
+## Building
+
+To build locally run the following:
+
+```sh
+./mvnw clean package
+```
+
+* If you get a `Could not find or load main class org.apache.maven.wrapper.MavenWrapperMain` exception, upgrade your local `wget`
+## Running Locally
+
+### Prerequisite
+
+Before starting locally you will need an instance of either Apache Cassandra&reg;. The easiest way to do this is with a
```

Comment:
Sentence feels like it trails off. "an instance of either C*....". Should there be an "or"?